### ISSUE TITLE
Drop IE9 & IE10 support

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#840]
Consider dropping IE9 & IE10 support

**Comment by stephenmathieson** [comment]
Big 👍 for me. As discussed on Slack, [IE9 only accounts for ~0.9% of all Internet traffic](https://en.wikipedia.org/wiki/Internet_Explorer_9), and [IE9 and IE10 combined only account for ~4% of all screenreader traffic](https://webaim.org/projects/screenreadersurvey7/#browsers). There is very little to continue to support them based on this data.

**Comment by stephenmathieson** [comment]
@dylanb that data is available publicly from multiple sources (including the source I provided in https://github.com/dequelabs/axe-core/issues/840#issuecomment-380891691). I do not think we need Marcy to talk to anyone to get it.

**Comment by dylanb** [comment]
that WebAIM data is very skewed and does not represent the audience we are actually interested in. We are interested in developers and particularly developers working for large organizations - like government, corporations, universities etc.

For many various reasons, the data we are looking for is not necessarily publicly available.

**Comment by stephenmathieson** [comment]
@dylanb Gotcha, okay. For my (and others' understanding), could you elaborate on why the globally accepted browser usage statistics (e.g. statcounter) are not acceptable here? It seems (IMO) that between the webaim and statcounter data we should be able to make a fairly educated decision here.

**Comment by stephenmathieson** [comment]
For those who reacted with a :-1:, can you please provide your reasoning? Why do you think axe-core should support IE9/10?

**Comment by WilcoFiers** [comment]
We've explored this and have decided that for axe-core 4.0 IE9 and IE10 will no longer be supported. We will continue to support IE11 for axe-core 4.x. Microsoft still supports it too for Windows 10, and it is something we know we have continued use for, for the foreseeable future.

**Comment by emilyrohrbough** [comment]
> Microsoft still supports it too for Windows 10, 

Do you mean Microsoft is maintaining a version of axe-core for IE 10 support?

**Comment by stephenmathieson** [comment]
> Microsoft still supports it too for Windows 10

Yeah, this makes sense. I assumed we'd keep IE11 around until Microsoft drops it. My concerns are for IE9/10.

IIRC (_part of_) the plan for 4.0 is to drop the bundled polyfills. Are there other breaking changes that would prevent axe from working in IE?

**Comment by emilyrohrbough** [comment]
If the change is to just drop the bundled polyfills, would it be possible add instructions on the polyfills needed to use axe-core in IE 9 & 10 and consumers could add this themselves to run in these browser?

**Comment by stephenmathieson** [comment]
> would it be possible add instructions on the polyfills needed to use axe-core in IE 9 & 10

Yes, absolutely.

**Comment by emilyrohrbough** [comment]
Awesome! I think that'd be a win-win for everyone then!

**Comment by WilcoFiers** [comment]
We're making this change for axe-core 4.0. We'll have to look at what polyfills we're taking out, we'll document what needs to be added in to run axe-core with IE 9 / IE 10.
